decred/decred/dcr/constants.py

```diff
@@ -5,12 +5,12 @@
 Just some constants.
 """
 
-import os
+from pathlib import Path
 
 
-PYDECRED_PACKAGEDIR = os.path.dirname(os.path.realpath(__file__))
-FAVICON = os.path.join(PYDECRED_PACKAGEDIR, "favicon-32x32.png")
-LOGO = os.path.join(PYDECRED_PACKAGEDIR, "logo.svg")
+ASSETS_DIR = Path(__file__).resolve().parent / "assets"
+FAVICON = ASSETS_DIR / "favicon-32x32.png"
+LOGO = ASSETS_DIR / "logo.svg"
```

Comment:
This is a breaking change, and indeed it does break the UI. These are expected to be strings in some contexts. 

https://github.com/teknico/tinydecred/blob/8e640f44c57d521954e282cde3f1bf7fd6e60cf6/tinywallet/tinywallet/app.py#L83

`Path` works in most Python STL contexts because it inherits `os.PathLike`, but filepaths are used other places as well. Also, since these are constants, the user should have little reason to prefer `Path` to string since there's no reason to manipulate the path. 

Unless there is a clear advantage to using `Path` that I'm not seeing, I suggest reverting this change. 

I do like the name `ASSETS_DIR` better.

Reply:
Ouch, I should have checked this one more thoroughly. Reverting to `os.path` while keeping the `ASSETS_DIR` name.

Reply:
It was worth the discussion. Plus, while investigating I found an unrelated bug.